statsmodels/tsa/tests/test_arima_process.py

```diff
@@ -155,8 +173,8 @@ def test_armafft():
             arma = ArmaFft(ar, ma, 20)
             ac1 = arma.invpowerspd(1024)[:10]
             ac2 = arma.acovf(10)[:10]
-            assert_almost_equal(ac1, ac2, decimal=7,
-                                err_msg='acovf not equal for %s, %s' % (ar, ma))
+            assert_allclose(ac1, ac2, atol=1e-15,
+                            err_msg='acovf not equal for %s, %s' % (ar, ma))
```

Comment:
Are the actual values of ac1 and ac2 big enough that atol=1e-15 is decently tight?